glue/core/data_factories/gridded.py

```diff
@@ -25,12 +25,19 @@ def is_fits(filename):
         return False
 
 
-def gridded_data(filename, format='auto', **kwargs):
+def gridded_data(filename, format='auto', wcs_ext=0, **kwargs):
```

Comment:
BTW, the format keyword is not really used in the function: it is just printed if the format is unknown, but the docstring tells it should override the format detection, which is not true.